doc/releases/changelog-0.36.0.md

```diff
@@ -215,23 +215,46 @@
 
 <h4>Simulate mixed-state qutrit systems 3️⃣</h4>
 
-* Functions `measure_with_samples` and `sample_state` have been added to the new `qutrit_mixed` module found in
- `qml.devices`. These functions are used to sample device-compatible states, returning either the final measured state or value of an observable.
+* Mixed qutrit states can now be simulated with the `DefaultQutritMixed` 
+  device (shortname: `"default.qutrit.mixed"`).
```

Comment:
```suggestion
* Mixed qutrit states can now be simulated with the `"default.qutrit.mixed" device.
```
Most users won't interact with the `DefaultQutritMixed` class but instead load it through `qml.device`, so probably makes sense to just mention that.